globus_cli/services/transfer/bookmark/locate.py

```diff
@@ -0,0 +1,49 @@
+from uuid import UUID
+
+import click
+
+from globus_sdk.exc import TransferAPIError
+from globus_cli.safeio import safeprint
+from globus_cli.parsing import common_options
+from globus_cli.services.transfer.helpers import get_client
+
+
+@click.command('locate', help='Output <UUID>:<path> of a bookmark; useful in '
+                              'a subshell as input to another command')
+@common_options
+@click.argument('bookmark_id_or_name', required=True)
+def bookmark_locate(bookmark_id_or_name):
+    """
+    Executor for `globus transfer bookmark locate`
+    """
+
+    client = get_client()
+
+    # leading/trailing whitespace doesn't make sense for UUIDs and the Transfer
+    # service outright forbids it for bookmark names, so we can strip it off
+    bookmark_id_or_name = bookmark_id_or_name.strip()
```

Comment:
I put that in there because I figured a very common case for this might be taking user input for a bookmark name, and using a bookmark name would *necessitate* quoting the user input (e.g. `globus transfer bookmark locate "$someUserInput"`), and I thought it would be super if the CLI did the right thing.

*But*, that said, maybe this is over-robust/over-polite *or* maybe we should refactor thing kind of thing into parameter type that knows to do stripping. Or maybe that's overengineering. Either way, if you don't want it I'll drop it out.